platform_maps.json

```diff
@@ -0,0 +1,188 @@
+{
+  "_comment": "Platform mapping configuration for RomM MuOS App",
+  "_description": "Maps RomM platform slugs to local filesystem directory names",
+  "_usage": {
+    "es_folder_map": "EmulationStation mappings - array format: [folder_name, icon_name]",
+    "muos": "MuOS-specific directory mappings",
+    "spruceos": "SpruceOS-specific directory mappings"
+  },
+  "_priority": "CUSTOM_MAPS env var > this file > hardcoded defaults",
```

Comment:
This is a weird style, I would move this to the README or something (since you can't have comments in json), or use YAML